include/tvm/runtime/logging.h

```diff
@@ -395,10 +399,13 @@ class LogMessageVoidify {
 inline bool DebugLoggingEnabled() {
   static int state = 0;
   if (state == 0) {
-    if (auto var = std::getenv("TVM_LOG_DEBUG")) {
-      if (std::string(var) == "1") {
+    if (const char* var = std::getenv("TVM_LOG_DEBUG")) {
+      std::string var_str(var);
+      if (var_str == "1" || var_str.rfind("1;", 0) == 0) {
```

Comment:
seems like there are multiple type information get put into TVM_LOG_DEBUG, this would make the use of such environment variable become trivial and complicated, please avoid such usage.

Reply:
Will address in your comment below.